src/zarr/core/group.py

```diff
@@ -747,7 +1223,9 @@ async def contains(self, member: str) -> bool:
             return True
 
     async def groups(self) -> AsyncGenerator[tuple[str, AsyncGroup], None]:
-        async for name, value in self.members():
+        members = [x async for x in self.members()]
+        # async for name, value in self.members():
+        for name, value in members:
```

Comment:
Why was this needed?

Reply:
Leftover from debugging I think. Removing.